### ISSUE TITLE
simple sendDTMF is not sending out the tone

### ISSUE THREAD

**Issue by robsonhermes** [onsip/SIP.js#773]
**Describe the bug**
When I call simple.sendDTMF(1), it breaks with "error tone is not defined". On a second attempt, it logs that it tried to send the dtmf tone, but nothing happens. Apparently not sent out.

I am dialing 1und1 public number 07219605727 to test sending dtmf tones.

**Logs**
Chrome console
![image](https://user-images.githubusercontent.com/26451017/74353273-f69d1680-4db9-11ea-9e4f-d1cde81492cc.png)

**To Reproduce (if possible)**
1. Start a Simple,
2. Call 1und1 number:  07219605727
3. Try to send a dtmf tone

**Expected behavior**
DTMF tone should be sent out

**Observed behavior**
On first attempt, an error saying "tone is not defined".
On second attempt, just a log in console "sip.simple | Sending DTMF tone: 1"

**Environment Information**
 - FreeSWITCH version: 1.8.5-6-31281a0bf1~64bit
 - Chrome 80.0.3987.87
 - sip-0.15.10.min.js

**Comment by robsonhermes** [comment]
Nevermind about the error, it was my bad on another part of my test. I was doing something on dtmf event and there was a bug there.
However, after fixing my typo there, still, no dtmf it sent out.

**Comment by EduardoSellanes** [comment]
@robsonhermes ... have you been able to try that? ^^

**Comment by robsonhermes** [comment]
@EduardoSellanes  hey, sorry about the delay. I still am trying to make my way on the (complete) version of SIP.js, but it's a bit hard of the recent changes in the api/interfaces.

Anyway, I just tried with your commit. Apparently the api is "trying to send it out" cause I can see it in the logs, but nothing really happens. Now I am wondering if this is some sort of compatibility problem with our FreeSWITCH.

I attached the console logs just in case. I dialed a number and tried to send DTMF 1 two times, but nothing really happened.
I just replaced my real servers with XXX.de in the log.

Hopefully I am not doing any stupid rookie mistake.

[SIP log.txt](https://github.com/onsip/SIP.js/files/4318982/SIP.log.txt)
